bpf/bpf_lxc.c

```diff
@@ -69,7 +69,9 @@ static inline int ipv6_l3_from_lxc(struct __sk_buff *skb,
 				   struct ipv6_ct_tuple *tuple, int l3_off,
 				   struct ipv6hdr *ip6, __u32 *dstID)
 {
+#ifdef ENABLE_ROUTING
 	union macaddr router_mac = NODE_MAC;
+#endif
```

Comment:
Small nit, feel free to ignore: we could potentially reduce scope as in https://github.com/cilium/cilium/blob/master/bpf/bpf_lxc.c#L115 which also consolidates into single ifdef.